Libraries/Components/WebView/WebView.android.js

```diff
@@ -127,13 +134,25 @@ var WebView = React.createClass({
       domStorageEnabled = this.props.domStorageEnabledAndroid;
     }
 
+    var htmlProp, urlProp, sourceProp;
+    if(this.props.html) {
+      console.warn('html is deprecated. Use source instead.');
+      htmlProp = this.props.html;
+    } else if(this.props.url) {
+      console.warn('url is deprecated. Use source instead.');
```

Comment:
I'd prefer a bit informative warnings. When I see YellowBox with this warning, it'll not be obvious to me regarding where it's coming from and what to do.

Some thing like,

`WebView: url prop is deprecated. Use the source prop instead.`